modules/tensor_mechanics/doc/content/source/materials/ADLAROMCreepStressUpdate.md

```diff
@@ -0,0 +1,168 @@
+polynomial of degree $i$, evaluated for input $x$, where the range of $x$ has been normalized to the
+$\left[-1,\ 1\right]$ interval. The model form includes polynomial terms of each input parameter and all interaction terms thereof, i.e.,
+\begin{equation}
+	Output\ \sim \sum_{w=0}^{deg}{\ldots\sum_{z=0}^{deg}{\alpha_{w\ldots z}P_w\left(x_w\right)\ldots}}P_z\left(x_z\right),
+\end{equation}
+where $deg$ is the maximum degree of polynomial to be used in the model, $\alpha_{w\ldots z}$ is the regression coefficient for the term formed from the
+ product of $w$-th degree polynomial of input $x_w$, etc.
+
+Three output values are required from the ROM: $\dot{\epsilon}_{vm}$, ${\dot{\rho}}_{cell}$, and
+${\dot{\rho}}_W$. While the strain rate is the desired output for predicting creep behavior, The
+defect density rates are also needed, as they affect the evolution of the creep rate over time
+through their own evolution. An entirely independent regression model is employed to predict each of
+the three outputs required. For example, the regression model for $\dot{\epsilon}_{vm}$ is shown, i.e.,
+\begin{equation}
+  {\dot{\varepsilon}}_{vm}\ ~\ \sum_{n_\varepsilon}^{deg}\sum_{n_{\rho_{cell}}}^{deg}\sum_{n_{\rho_W}}^{deg}\sum_{n_\sigma}^{deg}\sum_{n_T=0}^{deg}{\alpha_{n_\varepsilon n_{\rho_{cell}}n_{\rho_W}n_\sigma n_T}^{{\dot{\varepsilon}}_{vm}}P_{n_\varepsilon}\left(\varepsilon_{vm}\right)P_{n_{\rho_{cell}}}\left(\rho_{cell}\right)P_{n_{\rho_W}}\left(\rho_W\right)P_{n_\sigma}\left(\sigma_{vm}\right)P_{n_T}\left(T\right)\ }
+\end{equation}
+where each $n_{sub}$ is an index corresponding to the degree of the Legendre polynomial of the input parameter in the subscript. The coefficients are denoted
+as specific to the output $(\dot{\epsilon}_{vm})$ with a superscript. Additionally, mappings are
+used to convert from engineering input units to terms less susceptible to error propagation for
+informing the ROM. All inputs and outputs are handled internally such that the use of the ROM is
+unchanged by the inclusion of these mappings.
+
+The coefficients, $\alpha$, are evaluated by fitting the ROM to a database of results from the VPSC
+simulations using the physics-based constitutive laws. The maximum degree of the polynomial terms is
+selected to maximize fidelity to the data while avoiding an overfitting of the data, where
+overfitting is indicated by a stark difference in the regression fit to training and testing data.
+To validate the obtained regression coefficient values, initial conditions are given to the ROM and
+VPSC, and the resulting simulations are compared.  
+
+### Implementation
+
+While `ADLAROMCreepStressUpdate` contains the necessary algorithms contained to evaluate the ROM,
+the actual data that comprises the ROM is contained in a separate UserObject that inherets from
+`LAROMData`. The data contained in `LAROMData` is the maximum and minimum values used to calibrate
```

Comment:
Link directly to the LAROMData classes here?

Reply:
These classes will be folded into the main class.